packages/lesswrong/lib/collections/chapters/callbacks.js

```diff
@@ -30,3 +32,38 @@ function ChaptersEditHTMLSerializeCallback (modifier, chapter) {
 }
 
 addCallback("chapters.edit.sync", ChaptersEditHTMLSerializeCallback);
+
+async function ChaptersEditCanonizeCallback (chapter) {
+  const posts = await Sequences.getAllPosts(chapter.sequenceId)
+  const sequence = await Sequences.find({_id:chapter.sequenceId}).fetch()[0]
```

Comment:
I am surprised this even works. My expectation would be that you try to reference `[0]` of a promise, and then try to await it.